packages/helium-admin-cli/src/backfill-mobile-deployment-infos.ts

```diff
@@ -0,0 +1,262 @@
+      ...accountInfo,
+    }));
+
+  const ixs = (
+    await Promise.all(
+      accountInfosWithPk.map(async (acc) => {
+        if (acc.data) {
+          let correction: {
+            location?: anchor.BN;
+            deploymentInfo?: MobileDeploymentInfoV0;
+          } = {};
+
+          const decodedAcc: MobileHotspotInfo = hem.coder.accounts.decode(
+            "MobileHotspotInfoV0",
+            acc.data as Buffer
+          );
+
+          const deploymentInfoMissing =
+            !decodedAcc.deploymentInfo && hasDeploymentInfo(acc.wifiInfo);
+
+          const deploymentInfoChanged = !deepEqual(
+            decodedAcc.deploymentInfo?.wifiInfoV0,
+            acc.wifiInfo.deploymentInfo
+          );
+
+          const locationMissing = !decodedAcc.location && acc.wifiInfo.location;
+
+          if (deploymentInfoMissing || deploymentInfoChanged) {
+            correction = {
+              ...correction,
+              deploymentInfo: {
+                wifiInfoV0: {
+                  antenna:
+                    // db record is source of truth for antenna
+                    acc.wifiInfo.deploymentInfo.antenna ||
+                    decodedAcc.deploymentInfo?.wifiInfoV0?.antenna ||
+                    0,
+                  elevation:
+                    decodedAcc.deploymentInfo?.wifiInfoV0?.elevation ||
+                    acc.wifiInfo.deploymentInfo.elevation ||
+                    0,
+                  azimuth:
+                    decodedAcc.deploymentInfo?.wifiInfoV0?.azimuth ||
+                    acc.wifiInfo.deploymentInfo.azimuth ||
+                    0,
+                  mechanicalDownTilt:
+                    decodedAcc.deploymentInfo?.wifiInfoV0?.mechanicalDownTilt ||
+                    acc.wifiInfo.deploymentInfo.mechanicalDownTilt ||
+                    0,
+                  electricalDownTilt:
+                    decodedAcc.deploymentInfo?.wifiInfoV0?.electricalDownTilt ||
+                    acc.wifiInfo.deploymentInfo.electricalDownTilt ||
+                    0,
+                },
+              },
+            };
+          }
+
+          if (locationMissing) {
+            correction = {
+              ...correction,
+              location: acc.wifiInfo.location,
+            };
+          }
+
+          if (Object.keys(correction).length > 0) {
+            return await hem.methods
+              .tempBackfillMobileInfo({
+                location: correction.location || null,
+                deploymentInfo: correction.deploymentInfo || null,
+              })
+              .accounts({
+                payer: wallet.publicKey,
+                mobileInfo: acc.pubkey,
+              })
+              .instruction();
+          }
+        }
+      })
+    )
+  ).filter(truthy);
+
+  console.log(`Total corrections needed: ${ixs.length}`);
+  if (commit) {
+    try {
+      for (const c of chunks(ixs, 250)) {
+        await batchParallelInstructionsWithPriorityFee(provider, c);
```

Comment:
Instead of doing this in chunks, you can do this:

```
const transactions = await batchInstructionsToTxsWithPriorityFee(
    provider,
    instructions,
    {
      useFirstEstimateForAll: true,
    }
  );

  await bulkSendTransactions(provider, transactions, console.log);
  ```

It'll make it use waaaay less RPC since it won't simulate every tx to get its compute, and get an individual priority fee.